base/glibc-compatibility/glibc-compatibility.c

```diff
@@ -9,14 +9,25 @@ extern "C" {
 #endif
 
 #include <pthread.h>
-
+#include <unistd.h>
+#include <limits.h>
+
+/* Size of static TLS block.  */
+extern size_t __static_tls_size;
+/* This function is internal (it has a GLIBC_PRIVATE) version, but it
```

Comment:
This comment was copy-pasted verbatim* from https://public-inbox.org/libc-alpha/9f80a56f-8067-3c9d-6887-ab2077b208b7@redhat.com/T/

It cannot be done without violating the public license.
Should be rewritten or need to add reference, authorship and attribution and check the license compatibility.

\* it was suspected because of double whitespaces. I know that no one working on ClickHouse codebase would dare to write double whitespaces.